src/qt/coincontroldialog.cpp

```diff
@@ -165,6 +165,7 @@ void CoinControlDialog::setModel(WalletModel *_model)
         updateView();
         updateLabelLocked();
         CoinControlDialog::updateLabels(_model, this);
+        UpdateAmountHeader(_model);
```

Comment:
Why the extra roundtrip via `UpdateAmountHeader()` instead of just setting the header item here? Could use a comment to make it more obvious whats happening here.

Reply:
Maybe I could rename it to something like `UpdateAmountDisplayUnit()`? I realize that `UpdateAmountHeader()` is kind of vague.

Reply:
Agree with @jonasschnelli: just setting the header will do.